examples/Cargo.toml

```diff
@@ -195,3 +199,7 @@ path = "video_player/main.rs"
 [[bin]]
 name = "virtual_methods"
 path = "virtual_methods/main.rs"
+
+[[bin]]
+name = "request_no_blocking_main_thread"
+path = "request_no_blocking_main_thread/main.rs"
```

Comment:
Keep the file ordered please. Also maybe just name it `tokio` ? as it basically shows how to integrate tokio in your gtk-rs app.